webpack.config.js

```diff
@@ -474,6 +474,9 @@ if (IS_UI_DEV_ONLY) {
         target: 'https://sentry.io',
         secure: false,
         changeOrigin: true,
+        headers: {
```

Comment:
The comment in Line 460 should be updated because it is possible to write to the API now.